src/status_im2/navigation/roots.cljs

```diff
@@ -148,10 +139,19 @@
   []
   ;;TABS
   (merge (old-roots)
+         ;;INTRO (onboarding carousel)
+         {:intro-stack
```

Comment:
Could you please explain what's the way to add more screens to this `:intro-stack`? I guess we should add them into that `:children` vector, but I'm not sure.

Thanks!

Reply:
as far as I understand we should add them to src/status_im2/navigation/screens.cljs and then have a button that navigates to that screen. @Parveshdhull, that's how it should work,right?

Reply:
Yes, we should navigate to them with buttons.
Multiple children are used for pushing several screens at once while changing root.
For ex: After login instead of showing the shell screen, directly show chat screen.